examples/react-redux/package.json

```diff
@@ -0,0 +1,48 @@
+{
+    "name": "todomvc-react-redux",
+    "version": "1.0.0",
+    "description": "TodoMVC app written with React-Redux.",
+    "private": true,
+    "engines": {
+        "node": ">=18.13.0",
+        "npm": ">=8.19.3"
+    },
+    "scripts": {
+        "build": "webpack --config webpack.prod.js",
+        "dev": "webpack serve --open --config webpack.dev.js",
+        "serve": "http-server ./dist -p 7002 -c-1 --cors",
+        "test": "jest"
+    },
+    "devDependencies": {
+        "@babel/core": "^7.21.0",
+        "@babel/preset-env": "^7.20.2",
+        "@babel/preset-react": "^7.18.6",
+        "@testing-library/jest-dom": "^5.16.5",
+        "@testing-library/react": "^12.1.5",
+        "babel-loader": "^9.1.2",
+        "css-loader": "^6.7.3",
+        "css-minimizer-webpack-plugin": "^4.2.2",
+        "eslint-plugin-react": "^7.32.2",
+        "html-webpack-plugin": "^5.5.0",
+        "http-server": "^14.1.1",
+        "jest": "^29.4.3",
+        "jest-environment-jsdom": "^29.4.3",
+        "mini-css-extract-plugin": "^2.7.2",
+        "style-loader": "^3.3.1",
+        "webpack": "^5.75.0",
+        "webpack-cli": "^5.0.1",
+        "webpack-dev-server": "^4.11.1",
+        "webpack-merge": "^5.8.0"
+    },
+    "dependencies": {
+        "classnames": "^2.2.5",
+        "prop-types": "^15.8.1",
+        "react": "^16.14.0",
+        "react-dom": "^16.14.0",
+        "react-redux": "^7.2.6",
```

Comment:
Same for react-redux, the latest version is 8.1.2 as of the time of this review

Reply:
Same as comment above, 7.2.6 was the most popular version at the time we chose what version to upgrade to.